### DISCUSSION TITLE
Umask with podman exec

### DISCUSSION THREAD

**Discussion by Joniteus** [containers/podman#19712] [Q&A]
Hi,

I have set a new umask value to my containers.conf (0017) and now my containerized application writes it's log files with the permissions I want it to write them. BUT! As I was setting it up and tried to test the propagation of the settings by entering the running container with 'podman exec' the umask value was the default one (0022) instead of what was visible in the containers.conf and podman inspect container.

So the question follows. Why the umask setting is different in these cases?

**Comment by rhatdan** [comment]
It should not be exec and run should have the same umask.

**Reply by Joniteus** [reply]
podman inspect app
...
               "Umask": "0017",
...

podman exec -it app bash 
jetty@app :~$ umask -p

umask 0022

Container base image is  jetty:9.4.48-jre8-openjdk. I wonder if it's because of that.

podman --version
podman version 4.4.1
